app/components/info-cards/playlist-card.tsx

```
@@ -1,23 +1,83 @@
   return (
-    <GeneralCard
-      img={{ src: thumbnailurl || '/images/playlist-default.png', alt: `Playlist ${title}` }}
-      title={title}
-      subtitle={description}
-      href={`/playlists/${id}`}
-      type="playlist"
-    />
+    <div 
+      className="vertical-card rounded-lg bg-[--md-sys-color-outline-variant] flex flex-col items-center justify-start overflow-hidden w-full max-w-[280px] sm:max-w-[200px] h-full cursor-pointer" 
+      onClick={() => router.push(`/playlists/${id}`)}
+    >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)